yt_dlp/extractor/picarto.py

```diff
@@ -97,26 +98,39 @@ class PicartoVodIE(InfoExtractor):
     }, {
         'url': 'https://picarto.tv/videopopout/Plague',
         'only_matching': True,
+    }, {
+        'url': 'https://picarto.tv/ArtofZod/videos/772650',
+        'only_matching': True,
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
-        webpage = self._download_webpage(url, video_id)
+        data = traverse_obj(self._download_json(
+            'https://ptvintern.picarto.tv/ptvapi', video_id, query={
+                'query': '''{
+  video(id: "%s") {
+    id
+    title
+    file_name
+    video_recording_image_url
+    channel {
+      name
+    }
+  }
+}''' % (video_id),
+            }), ('data', 'video'))
```

Comment:
no point in making this non-fatal with `traverse_obj` if `title` and `netloc` are both fatal
```suggestion
        data = self._download_json(
            'https://ptvintern.picarto.tv/ptvapi', video_id, query={
                'query': '''{
  video(id: "%s") {
    id
    title
    file_name
    video_recording_image_url
    channel {
      name
    }
  }
}''' % video_id,
            })['data']['video']
```

Reply:
Thanks, fixed in latest diff.